test/ConfCrypt/Commands/Tests.hs

```diff
@@ -68,12 +69,14 @@ bufferWriteProperties  = testGroup "Buffer write" [
         in either (const False)
                   (\ccf' -> fileContents ccf' == fc)
                   parseRes
-    ]
+    ]-}
 
 readTests :: TestTree
 readTests = testGroup "Read" [
     testCase "reading produces decrypted results" $ do
-        let testLines = parseConfCrypt "test file" testFile
+        let filePath = "testFile"
+        lines <- T.readFile filePath
```

Comment:
Does this no longer work using the default `testFile`?

Reply:
Yes, in my first implementation it was looking for a real file.
With last commit it has been restored.